### DISCUSSION TITLE
DS18B20 connected to ESP-01 not publishing on event

### DISCUSSION THREAD

**Discussion by thesmartalex** [arendst/Tasmota#14648] [Q&A] [answered]
Hello.
I have 8x DS18B20 sensors connected to a ESP-01 running tasmota. All the sensors measure correctly but they do not publish  on event (temp change) they are only publishing on the teleperiod. Is there a way to make them publish on event?
Thank you!

**Comment by ascillato** [comment, marked as answer]
Hi,

Yes, using rules. Full example in the docs at: https://tasmota.github.io/docs/Rules/#transmit-sensor-value-only-when-a-delta-is-reached
